### DISCUSSION TITLE
Movie Collections

### DISCUSSION THREAD

**Discussion by darqman** [croneter/PlexKodiConnect#1381] [Q&A] [answered]
Are movie collections supposed to sync over from Plex to Kodi? 
I so, not all of of mine are.

**Comment by croneter** [comment, marked as answer]
Yes they are synched. Kodi only supports a movie belonging to a single set/collection, however. If a movie belongs to several sets/collections on the Plex side of things, only one random set/collection will be synched.

Workaround: navigate to add-ons > video add-ons > PlexKodiConnect > (Name of your Plex library) > collections. You can set a Kodi widget accordingly

**Reply by darqman** [reply]
I must be doing something wrong then.  I have 21 collections in Plex but only 9 show in Kodi. My Alfred Hitchcock collection has 19 movies. The individual titles show in the library but there's not an Alfred Hitchcock collection or box set in Kodi.  Maybe it's the skin I'm using? I'm using Amber. None of the titles belong to multiple collections.

**Reply by croneter** [reply]
Try the Estuary skin. Are all collections on the same Plex library?

**Reply by darqman** [reply]
Yes, I have one Plex library called Movies.

**Reply by croneter** [reply]
Share 2 screenshots then please: the Plex Collections as well as the collections showing up in Kodi

**Reply by darqman** [reply]
Yes sir,

One screenshot shows 22 collections in Plex and the other is a picture of the screen in my theater with Kodi showing only 10 collections. Same library from both named "Movies".  I switched to the default skin but that didn't change anything. Not sure what's going on.

**Reply by darqman** [reply]
![kodi-collections](https://user-images.githubusercontent.com/69403536/110418498-4a317080-805d-11eb-9f7f-a4010b5ae233.jpg)
![plex-collections](https://user-images.githubusercontent.com/69403536/110418504-4dc4f780-805d-11eb-8128-1c9a9a6b3d8e.JPG)

**Reply by croneter** [reply]
Weird! Did you try to reset the Kodi database in the pkc database under Advanced? (Red text) Choose YES, then NO

**Reply by darqman** [reply]
I will try that tonight. However I have Kodi and PKC setup on 4 devices (2 Chromecasts w/Google TV) and 2 Dune-HD players and all 4 devices have the same issues. Some have have 10 collections, some have 12, but none have all 22. I'll report back after I reset the db.

**Reply by darqman** [reply]
Confirmed that resetting the DB in PKC did work!! Thank you!

**Reply by S1llyG1t** [reply]
> Yes they are synched. Kodi only supports a movie belonging to a single set/collection, however. If a movie belongs to several sets/collections on the Plex side of things, only one random set/collection will be synched.
> 
> Workaround: navigate to add-ons > video add-ons > PlexKodiConnect > (Name of your Plex library) > collections. You can set a Kodi widget accordingly

Sorry to bump this, but can't seem to find any other relevant information or fixes for this issue. I have upgraded from Kodi Matrix to Nexus, hoping that maybe Kodi would have addressed this. That does not appear to be the case, but hoping for any updated information.

I'm confused by the workaround above. Is that still supposed to work? If I follow that path assuming by (Name of your Plex Library) you meant like "Movies" for example. For me, the Collections there show the same as the movie-sets or sets widgets in the normal kodi ui.

If there truly is a workaround to see 1:1 collections from plex, I'd really love to know.
